src/renderer/components/+catalog/catalog-entity-detail-items/extension-catalog-entity-detail-items-registrator.injectable.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { pipeline } from "@ogre-tools/fp";
+import { getInjectable } from "@ogre-tools/injectable";
+import { forEach } from "lodash/fp";
+import { extensionRegistratorInjectionToken } from "../../../../extensions/extension-loader/extension-registrator-injection-token";
+import type { LensRendererExtension } from "../../../../extensions/lens-renderer-extension";
+import { catalogEntityDetailItemInjectionToken } from "./catalog-entity-detail-item-injection-token";
+
+const extensionCatalogEntityDetailItemsRegistratorInjectable = getInjectable({
+  id: "extension-catalog-entity-detail-items-registrator",
+
+  instantiate:
+    (di) => (extension: LensRendererExtension, installationCounter) => {
+      pipeline(
+        extension.catalogEntityDetailItems.map((registration, index) =>
+          getInjectable({
+            id: `catalog-entity-detail-item-${index}-from-${extension.sanitizedExtensionId}-instance-${installationCounter}`,
+
+            instantiate: () => ({
+              apiVersions: registration.apiVersions,
+              kind: registration.kind,
+
+              components: {
+                Details: registration.components.Details,
+              },
+
+              orderNumber: -registration.priority || -50,
```

Comment:
Because we changed it to `orderNumber`. Priority has double meaning; it doesn't tell whether items should be sorted in ascending or descending order. `orderNumber` does so. It also allows us to use `sortBy` from `lodash/fp`

Reply:
But then we have to remember to make them negative, could sort by increasing by default?

Reply:
No need, in extension API property name is still `priority` and it's positive, but in OpenLens (and hopefully in Extension API 2.0) property name is `orderNumber` which is also positive. Code that we are discussion about is the adapter which converts Extension API 1.0 to internal API.